calico_node/allocateipip/allocate_ipip_addr.go

```diff
@@ -213,7 +213,10 @@ func getIPIPEnabledPoolCIDRs(ctx context.Context, c client.Interface) []net.IPNe
 	var cidrs []net.IPNet
 	for _, ipPool := range ipPoolList.Items {
 		_, poolCidr, err := net.ParseCIDR(ipPool.Spec.CIDR)
-		log.WithError(err).Fatalf("Failed to parse CIDR: '%s'", ipPool.Spec.CIDR)
+		if err != nil {
+			log.WithError(err).Fatalf("Failed to parse IPPool CIDR '%s' for IPPool '%s'", ipPool.Spec.CIDR, ipPool.Name)
```

Comment:
Don't need `IPPool` twice in the same log - just `Failed to parse CIDR '%s' for IPPool '%s'`